packages/server/src/util/test-utils/data/observation-generator.ts

```diff
@@ -13,6 +14,18 @@ export function observationTestData(userEdipi: string, unitName: string, reportS
     observation.timestamp = new Date(currDate.toISOString());
     observation.documentId = `DocumentId_${userEdipi}_${reportSchema.id}_${count}`;
     observation.edipi = userEdipi;
+    const customCol: CustomColumns = {
+      Details: {
+        TalkToSomeone: true,
+        TemperatureFahrenheit: 102,
+        Symptoms: 'cough, fever',
+      },
+      Category: {
+        Keyword: 'Medium',
```

Comment:
I think category shouldn't be coming in with `Keyword`.  I believe that's something specific to elasticsearch.  Instead it should probably be `Category: 'Medium',`.

Reply:
good catch thanks!